src/iconsManifest/supportedExtensions.ts

```diff
@@ -2942,6 +2942,40 @@ export const extensions: IFileCollection = {
       light: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'mivascript',
+      extensions: ['mv'],
+      languages: [languages.mv],
+      format: FileFormat.svg
+    },
+    {
+      icon: 'mvt',
+      extensions: ['mvt'],
+      languages: [languages.mvt],
+      format: FileFormat.svg
+    },
+    {
+      icon: 'mvtcss',
+      extensions: [],
+      extensionsGlob: [
+        'mvt',
+        '*.css.mvt',
+        'css/*.mvt'
+      ],
+      languages: [languages.mvtcss],
```

Comment:
If you declare a language id, there's no need to use extensions at all. Usually, a language id is then managed via an extension. In this case, the Miva extension seems to be declaring some language Ids and its related extensions. That's all that we need.

Reply:
These are the settings we use internally to get the icons showing for mvtjs/css files since they technically share the same `.mvt` extension. The only way we know they should be `mvtjs`/`mvtcss` syntax and associated icon is to check if they are contained within the `js` or `css` folder in our template directory.

```
"vsicons.associations.files": [
	{
		"icon": "mivascript",
		"extensions": [
			"mv"
		],
		"languages": [
			{
				"ids": "mv",
				"defaultExtension": "mv"
			}
		],
		"format": "svg"
	},
	{
		"icon": "mvt",
		"extensions": [
			"mvt"
		],
		"languages": [
			{
				"ids": [
					"mvt"
				],
				"defaultExtension": "mvt"
			}
		],
		"filename": true,
		"format": "svg"
	},
	{
		"icon": "mvtjs",
		"extensions": [
			"js/*.mvt"
		],
		"languages": [
			{
				"ids": [
					"mvtjs"
				],
				"defaultExtension": "mvt"
			}
		],
		"format": "svg"
	},
	{
		"icon": "mvtcss",
		"extensions": [
			"css/*.mvt"
		],
		"languages": [
			{
				"ids": [
					"mvtcss"
				],
				"defaultExtension": "mvt"
			}
		],
		"format": "svg"
	}
]
```

What would be the correct translation of these settings to the `supportedExtensions.ts` file version?

Reply:
Given that you have an extension and you have added the language ids, there's no need to define the extensions. Those will used from the miva extension.